package.json

```diff
@@ -25,19 +26,28 @@
   },
   "dependencies": {
     "@types/node": "^13.7.4",
+    "@types/pg": "^7.14.3",
```

Comment:
Would typically agree since this lib should not be exporting public types from `@types/pg`, however:
* this repo is intended to be a service/app (rather than a published lib), and in the docker build we will probably clean `devDependencies` from the final build
* the docker run command will probably use `ts-node` to start the process, and would fail if this package was stripped

I'm not convinced about that last point -- `ts-node` used to recommend not using in production, but has since changed, and more apps are using it. However, it does put a noticeable startup latency for the service...